src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -29,6 +29,16 @@ internal class RuntimeAssembly : Assembly
 
         #endregion
 
+        private class ManifestResourceStream : UnmanagedMemoryStream
+        {
+            private RuntimeAssembly _manifestAssembly;
```

Comment:
What about releasing the reference on explicit Dispose?

Reply:
Explicit Dispose does not guarantee that nobody is accessing the memory anymore

Reply:
fair enough, it'd have to invalidate the object state (maybe v2)